apps/snitch_core/lib/core/tools/helpers/taxonomy.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Snitch.Tools.Helper.Taxonomy do
+  @moduledoc """
+  Provides helper funtions to easily create taxonomy.
+  """
+
+  alias Snitch.Domain.Taxonomy, as: TaxonomyDomain
+  alias Snitch.Data.Schema.{Taxon, Taxonomy}
+  alias Snitch.Repo
+
+  @doc """
+  Creates taxonomy from the hierarchy passed.
+
+  Structure of hierarchy should be in following format
+        {"Brands",
+          [
+            {"Bags", []},
+            {"Mugs", []},
+            {"Clothing",
+             [
+               {"Shirts", []},
+               {"T-Shirts", []}
+          ]}
+        ]}
+  """
+  @spec create_taxonomy({String.t(), []}) :: Taxonomy.t()
+  def create_taxonomy({parent, children}) do
```

Comment:
1. Why is the argument a tuple?
2. This function and `create_taxon/2` should make use of `Ecto.Multi`.
3. This can be sped up a bit by using `Repo.insert_all` for each level of the tree, but the current approach is good enough.